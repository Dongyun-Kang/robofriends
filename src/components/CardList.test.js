import { shallow } from "enzyme";
import React from "react";
import CardList from "./CardList";

it("expec to render Card component", () => {
  const mockRobots = [
    {
      id: 1,
      name: "John Snow",
      username: "JS",
      email: "john@gmail.com",
    },
  ];
  expect(shallow(<CardList robots={mockRobots} />)).toMatchSnapshot();
  expect(shallow(<CardList robots={mockRobots} />).length).toEqual(1);
});
