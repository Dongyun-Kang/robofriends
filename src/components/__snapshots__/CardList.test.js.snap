// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expec to render Card component 1`] = `ShallowWrapper {}`;
